projects/js-packages/publicize-components/src/store/selectors.js

```diff
@@ -62,6 +62,29 @@ export function getLinkedInDetails( { forceDefaults = false } = {} ) {
 	};
 }
 
+/**
+ * Returns a template for Instagram data, based on the first Instagram account found.
+ *
+ * @returns {{name: string; profileImage: string}} The Instagram account data.
+ */
+export function getInstagramDetails() {
+	const connection = getConnections().find(
+		( { service_name } ) => 'instagram-business' === service_name
+	);
+
+	if ( connection ) {
+		return {
+			name: connection.display_name,
```

Comment:
Hopefully we can update this to the username soon.

Reply:
Yeah.